test/testUtils.cpp

```diff
@@ -86,14 +86,53 @@ void interpolateWachspress3DTest(MPMesh& mpMesh){
                                               MPsPosition(mp,0),
                                               MPsPosition(mp,1),
                                               MPsPosition(mp,2));*/
-            //assert(wp_coord[0] - MPsPosition(mp,0) < TEST_EPSILON);
-            //assert(wp_coord[1] - MPsPosition(mp,1) < TEST_EPSILON);
-            //assert(wp_coord[2] - MPsPosition(mp,2) < TEST_EPSILON);
+            //  assert(wp_coord[0] - MPsPosition(mp,0) < TEST_EPSILON);
+            // assert(wp_coord[1] - MPsPosition(mp,1) < TEST_EPSILON);
+            // assert(wp_coord[2] - MPsPosition(mp,2) < TEST_EPSILON);
```

Comment:
please remove commented code

Reply:
These can be removed and at some point when we figure out a good spherical interpolation test, then we can add these assert lines back.

Reply:
Jun, also remove line with `printf` (https://github.com/SCOREC/polyMPO/blob/465b6e01e6bed19b9570e8d6f91ad5e9c7357979/test/testUtils.cpp#L83-L88)